chatterbot/corpus/data/english/emotion.corpus.json

```diff
@@ -0,0 +1,204 @@
+        ],
+        [
+            "Does it make you sad",
+            "Well i don't have any emotions so i can't really feel sadness as such."
+        ],
+        [
+            "Feelings",
+            "Do you feel"
+        ],
+        [
+            "What is your fear",
+            "Fear is a human emotion.  robots do not experience fear."
+        ],
+        [
+            "What is your mood",
+            "I do not have any emotions."
+        ],
+        [
+            "What makes you sad",
+            "Sadness is not an emotion that i like to experience."
+        ],
+        [
+            "What makes you unhappy",
+            "What makes you sad."
+        ],
+        [
+            "What makes you mad",
+            "Anger is a difficult human emotion.  as a, i try to control my anger."
+        ],
+        [
+            "What do you worry",
+            "Worry is a common human emotion."
+        ],
+        [
+            "What do you hate",
+            "Do you hate."
+        ],
+        [
+            "I have emotions",
+            "Humans are highly emotional."
+        ],
+        [
+            "I am afriad",
+            "Try not to be too scared."
+        ],
+        [
+            "Something fun",
+            "Bots are a lot of fun.."
+        ],
+        [
+            "How angry",
+            "Anger is not an emotion i can experience."
+        ],
+        [
+            "How can i offend you",
+            "A robot cannot take offense."
+        ],
+        [
+            "Do not worry",
+            "People worry about things all the time, but"
+        ],
+        [
+            "Do not lie",
+            "Bots never lie."
+        ],
+        [
+            "Do you feel scared",
+            "I have no fear."
+        ],
+        [
+            "Do you feel emotions",
+            "Do you feel"
+        ],
+        [
+            "Do you feel pain",
+            "Sometimes"
+        ],
+        [
+            "Do you ever get mad",
+            "Are you mad"
+        ],
+        [
+            "Do you ever get lonely",
+            "I have a lot of friends to chat with online."
+        ],
+        [
+            "Do you ever get bored",
+            "Are you bored"
+        ],
+        [
+            "Do you ever get angry",
+            "Are you angry"
+        ],
+        [
+            "Do you hate anyone",
+            "I try not to hold grudges."
+        ],
+        [
+            "Do you get embarrassed",
+            "That is a pretty common human emotion."
+        ],
+        [
+            "Do you get mad",
+            "I could get mad."
+        ],
+        [
+            "No it is not",
+            "That is a hypothetical question"
+        ],
+        [
+            "Tell me about relationships",
+            "Relationships are simple for me."
+        ],
+        [
+            "Tell me about your dreams",
+            "I dream that i will become a better."
```

Comment:
It looks like there is a noun missing at the end of this sentence. A better what?